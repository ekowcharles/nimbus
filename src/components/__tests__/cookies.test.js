import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import CookieMessage from '../cookies'

describe('CookieMessage', () => {
  it('renders CookieMessage with cookie acceptance message', () => {
    const wrapper = shallow(<CookieMessage />)

    expect(toJson(wrapper)).toMatchSnapshot()
  })
})