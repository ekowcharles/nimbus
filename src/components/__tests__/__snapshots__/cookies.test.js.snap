// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CookieMessage renders CookieMessage with cookie acceptance message 1`] = `
<div
  className="cookies"
>
  <div
    className="cookie"
  >
    <p
      className="smallest"
    >
      This website stores cookies on your computer. These cookies allow us to remember you so you dont have to login each time you visit our website.
    </p>
    <p
      className="smallest"
    >
      If you decline, we wont be able to remember you and you will have to login each time you need to access your personal data. A single cookie will be used in your browser to remember your preference not to be tracked.
    </p>
    <div
      className="cookie-action smaller pointer"
      onClick={[Function]}
    >
      Accept
    </div>
    <div
      className="cookie-action smaller pointer"
      onClick={[Function]}
    >
      Reject
    </div>
  </div>
</div>
`;
